content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -103,6 +103,17 @@ The following methods exist for installing kubectl on macOS:
    sudo chown root: /usr/local/bin/kubectl
    ```
 
+   {{< note >}}
+   For more information about what PATH is, please refer to this [article](https://kb.iu.edu/d/acar).
+   
+   On MacOS, usually `/usr/local/bin/` will be included in the PATH.
+
+   You can run the following command to detect:
+   ```bash
+   es=$(echo $PATH | grep /usr/local/bin);if [[ -z "$es" ]]; then;echo "Detected that PATH does not contain /usr/local/bin";echo "Please refer to https://kb.iu.edu/d/acar, add /usr/local/bin to the PATH";exit 255;fi; echo "You can proceed to the next step";
+   ```
```

Comment:
This might close the terminal, surprising a reader. Let's give them a command to just _check_ it, that never has side-effects.